### PR TITLE
apply white menu

### PR DESCRIPTION
**Pull request by brindy**
Asana Issue URL: https://app.asana.com/0/inbox/392891325557408/531181521679102/531510529165908

## Description

Use a white menu instead of the jarring orange/red one.

Also fix bug where the menu doesn't disappear when tapping outside of it (on API 22 or lower).

Tested on API 22, 23 and 26 (emulator) and One Plus One (API 22).
 
## Steps to Test this PR:
1. Launch the app - open the menu. 
1. Check Back/forward look (and are) disabled.
1. Check Bookmarks button looks (and is) enabled.
1. Search for something and click a link.  
1. Open menu and back button should look enabled.
1. Tap back - should navigate back. 
1. Open menu - forward should look and be enabled.
1. Tap forward - should navigate forward
1. Open menu - tap bookmarks - should open bookmarks UI

### CODE REVIEW

File: app/src/main/java/com/duckduckgo/app/browser/BrowserPopupMenu.kt [hunk 1]
```diff
@@ -54,7 +54,7 @@ class BrowserPopupMenu : PopupWindow {
         val margin = 30
 
         fun inflate(layoutInflater: LayoutInflater): View {
-            return layoutInflater.inflate(R.layout.popup_window_brower_menu, null)
+            return layoutInflater.inflate(R.layout.popup_window_browser_menu, null)
```
Comment (subsymbolic) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserPopupMenu.kt]: Thanks for that!

File: app/src/main/java/com/duckduckgo/app/browser/BrowserPopupMenu.kt [hunk 2]
```diff
@@ -30,9 +30,9 @@ class BrowserPopupMenu : PopupWindow {
     constructor(layoutInflater: LayoutInflater, view: View = BrowserPopupMenu.inflate(layoutInflater))
             : super(view, WRAP_CONTENT, WRAP_CONTENT, true) {
 
-        if (SDK_INT <= 21) {
-            // popupwindow gets stuck on the screen on API 21 without a background color.
-            // Adding it however garbles the elevation so we cannot have elevation here
+        if (SDK_INT <= 22) {
+            // popupwindow gets stuck on the screen on API 22 (tested on 23) without a background
```
Comment (subsymbolic) [inline app/src/main/java/com/duckduckgo/app/browser/BrowserPopupMenu.kt]: Nice :)

File: app/src/main/res/layout/popup_window_browser_menu.xml [hunk 3]
```diff
@@ -36,7 +36,9 @@
             android:layout_weight="1"
             android:contentDescription="@string/back"
             android:onClick="onGoBackClicked"
-            android:src="@drawable/ic_arrow_back_24px" />
+            android:src="@drawable/ic_arrow_back_24px"
+            android:tint="@color/popup_menu_buttons_tint"
```
Comment (subsymbolic) [inline app/src/main/res/layout/popup_window_browser_menu.xml]: The tint was defined in the shared `BrowserIconMenuItem` style and point to an existing selectable drawable. Let's remove these duplicates and update `browser_menu_icon`. I think you may be able to delete the paleOrange color from colors now too.

Comment (brindy) [reply app/src/main/res/layout/popup_window_browser_menu.xml]: ah thanks! that explains why I couldn't see it.

**Comment by brindy** [comment]
@subsymbolic requested updated applied

**Pull request merged**